DOCS/man/options.rst

```diff
@@ -5438,6 +5437,7 @@ them.
 ``--correct-downscaling``
     When using convolution based filters, extend the filter size when
     downscaling. Increases quality, but reduces performance while downscaling.
+    Enabled by default.
 
     This will perform slightly sub-optimally for anamorphic video (but still
     better than without it) since it will extend the size to match only the
```

Comment:
a few lines below this it says
> Note: this option is ignored when using bilinear downscaling (the default).

needs to be changed too

Reply:
That comment is also not true on `gpu-next`, only on `gpu`.

Reply:
Fixed.